FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -1787,6 +2020,71 @@ components:
         - sourceId
         - description
         - timeOfBlocklist
+           
+    UnavailabilitySchedule:
```

Comment:
```suggestion
    RecurringSchedule:
```
How do we feel about just 'schedule' or 'RecurringSchedule'? Nothing about this needs to be specifically for unavailability scheduling, and this keeps it more open for future re-use

Reply:
If accepted, need to scrub for 'unavailability'-specific wording throughout this data structure

Reply:
Accepted your suggestion. thanks